main.cpp

```diff
@@ -1,167 +1,227 @@
+    
+    int fingers = 5;
+    int hands = 2;
+    int toes = 5;
+    
+    float speed = 60.f;
+    float heading = 10.f;
+    float direction = 1.f;
+    
+    bool hasPizza = false;
+    bool hasBeer = true;
+    bool hasSmile = true;
+    
+    double small = 0.00f;
+    double large = 1.00f;
+    double trouble = 0.5f;
+    
+    char R = 'R';
+    char Y = 'Y';
+    char A = 'A';
+    
+    ignoreUnused(number, fingers, hands, toes, speed, heading, direction, hasPizza, hasBeer, hasSmile, small, large, trouble, R, Y, A);
+  // passing each variable declared to the
+  // ignoreUnused() function
 }
 
 /*
  10 functions
  example:
  note: this example shows the result after completing steps 3-8
  */
-bool rentACar(int rentalDuration, int carType = 0)  //function declaration with random number of arguments, arbitrary number of arguments have default value
-{ 
-    ignoreUnused(rentalDuration, carType); //passing each function parameter to the ignoreUnused() function
-    return {}; //if your function returns something other than void, add 'return {};' at the end of it.
-} 
-
+bool rentACar(int rentalDuration, int carType = 0) // function declaration with random number of arguments, arbitrary number of arguments have default value
+{
+    ignoreUnused(rentalDuration, carType); // passing each function parameter to the ignoreUnused() function
+    return {}; // if your function returns something other than void, add 'return {};' at the end of it.
+}
+
 /*
  1)
  */
-
+bool makeMusic(int playGuiter = 1, int playBass = 2) 
+{  
+    ignoreUnused(playGuiter, playBass);
+    return {};
+}
 /*
  2)
  */
-
+float swim(bool inPool = true, bool inLake = false, bool inRiver = false) 
+{
+    ignoreUnused(inPool, inLake, inRiver);
+    return {};
+}
 /*
  3)
  */
-
+double writeCode(char numberOfLines = 'Lots', float quality = 0.1f)
+{  
+    ignoreUnused(numberOfLines, quality);
+    return {};
+}
 /*
  4)
  */
-
+int driveCar(int howFast = 160)
+{
+    ignoreUnused(howFast);
+    return {};
+}
 /*
  5)
  */
-
+char makeDinner(int numPeople = 5, int numPlates = 10, int numForks = 10)
+{
+    ignoreUnused(numPeople, numPlates, numForks);
+    return {};
+}
 /*
  6)
  */
-
+int paymentMethod(int ccNumber = 000000, int bankAcct = 000001, bool wireTransfer = true)
+{
+    ignoreUnused(ccNumber, bankAcct, wireTransfer);
+    return {};
+}
 /*
  7)
  */
-
+float frequency(int hz = 220, int cycles = 1, bool phase = true)
+{
+    ignoreUnused(hz, cycles, phase);
+    return {};
+}
 /*
  8)
  */
-
+bool isGood(bool taste = false)
+{
+    ignoreUnused(taste);
+    return {};
+}
 /*
  9)
  */
-
+double precision(float predict = 0.9f, float vectorX = 0.1f, float vectorY = 0.1f, float vectorZ = 0.1f)
+{
+    ignoreUnused(predict, vectorX, vectorY, vectorZ);
+    return {};
+}
 /*
  10)
  */
-
+char madeUpColor(char spectrum = 'foo', char band = 'bar')
```

Comment:
This function name does not conform to the programming guideline:
```
Name Things what they are.
Name Actions based on what they do
```
The name you chose reflects a property (noun) and does not contain a verb, which would reflect an action.
Functions perform work, so their names must contain a verb which describes the work being performed.
Examples of function names that contain verbs:
- `computeAValue()`
- `estimateDistanceTraveled()`
- `performDiagnostics()`
- [ ] revise